client/mysqldump.c

```diff
@@ -742,6 +746,14 @@ static void write_header(FILE *sql_file, char *db_name)
             "/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;\n",
             path?"":"NO_AUTO_VALUE_ON_ZERO",compatible_mode_normal_str[0]==0?"":",",
             compatible_mode_normal_str);
+    fprintf(sql_file,
+            "/*!50601 SELECT count(*) INTO @is_rocksdb_supported FROM"
+            " information_schema.SESSION_VARIABLES WHERE"
+            " variable_name='rocksdb_bulk_load' */;\n"
+            "/*!50601 SET @enable_bulk_load = IF (@is_rocksdb_supported,"
+            " 'SET SESSION rocksdb_bulk_load=1', 'SET @dummy = 0') */;\n"
```

Comment:
Hmm, perhaps, maybe a specifically named dummy or a SELECT 1; or something. Again, part of original patch that I really didn't consider much.